libs/ui/search/src/lib/results-list/results-list.component.html

```diff
@@ -5,13 +5,17 @@
   ></ng-container>
   <div *ngSwitchCase="layoutEnum.LIST" class="gap-4 p-4">
     <div class="pb-4" *ngFor="let record of records">
-      <gn-ui-record-preview-list [record]="record"></gn-ui-record-preview-list>
+      <gn-ui-record-preview-list
+        [record]="record"
+        (mdSelect)="mdSelect.emit($event)"
```

Comment:
To be honest I'm not sure if the `mdSelect` event brings any value; why not just do `(click)="mdSelect.emit(record.uuid)"` here, and get rid of all the emitter logic in `gn-ui-record-preview-list`?

Reply:
Yes should be way better than listening to the event in the `Preview` abstract component.
Thanks

Reply:
Actually it does not work because the `PreviewComponent` is a HREF that by default opens the `metadataUrl` link in an other tab.
My event was breaking that anyway. I'm wondering what is the best approach:
1. remove all the a-href  link stuff and propagate an event from `gn-ui-results-list`. In the application (or the webcomponent), get the event and put it in the state or manually open the link in another tab in javascript.
2. pass down a property to tells if we register to the event or not, if not the normal HTML href processes the click.

Reply:
having `<a>` elements with href pointing to a full metadata view is pretty nice I think, it makes sense in some cases and lets the browser handle things natively.

I think it might work with 2: we stop event propagation when the record is clicked if we want to intercept it